clients/web/src/router.js

```diff
@@ -25,63 +26,70 @@ export default new Router({
       pathToRegexpOptions: { strict: true },
     },
     {
-      path: '/:org/assess/:id/',
+      path: '/:org/create/',
+      name: 'assessmentCreate',
+      component: AssessmentCreate,
+      props: true,
+      pathToRegexpOptions: { strict: true },
+    },
+    {
+      path: '/:org/home/:slug/*/',
       name: 'assessmentHome',
       component: AssessmentHome,
       props: true,
       pathToRegexpOptions: { strict: true },
     },
     {
-      path: '/:org/assess/:id/intro/*/',
+      path: '/:org/assess/:slug/intro/*/',
       name: 'introPractices',
       component: IntroCurrentPractices,
       props: true,
       pathToRegexpOptions: { strict: true },
     },
     {
-      path: '/:org/assess/:id/content/*/',
+      path: '/:org/assess/:slug/*/',
```

Comment:
Why are you removing the `/content/` in URLs?

Reply:
I reverted this change.